palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -133,11 +68,25 @@ def replacements(self) -> dict:
         """
         Return a dictionary of replacements for the dbt template.
         """
-        return {
+        profile_strategy = self.determine_profile_strategy(Path.cwd())
+        replacements = dict()
+        if any(profile_strategy):
+            replacements = { 
+             "dbt_profile_host": profile_strategy[0],
+             "profile_volume_mount": ":".join(("${DBT_PROFILE_HOST}",
+                                                profile_strategy[1]))
+            } 
+            with Path(".env").open("a") as env_file:
+                env_file.write((f"DBT_PROFILE_HOST={profile_strategy[0]}\n"
```

Comment:
Writing the env file doesn't really belong here, this function should be responsible for building the replacements dict. Can we refactor?

Reply:
will do!